Sources/Fuzzilli/FuzzIL/Operations.swift

```diff
@@ -240,6 +240,20 @@ class CreateArrayWithSpread: Operation {
     }
 }
 
+class CreateTemplateLiteral: Operation {
+    // Stores the string elements of the temaplate literal
+    let literals: [String]
+
+    var numIdentifiers: Int {
+        return numInputs
+    }
+
+    init(literals: [String]) {
+        self.literals = literals
+        super.init(numInputs: literals.count > 0 ? literals.count - 1 : 0, numOutputs: 1, attributes: [.isVarargs, .isPure])
```

Comment:
Right, so the main reason I didn't want this to be `.isParametric` is because it will most likely mutate imported templates (which would mostly be valid JS snippets) and replace them with random strings and/or other template strings that may not be syntactically valid. These sort of mutations to the overall program would just error out and be discarded. Thoughts?

Reply:
Ok yeah that makes! So maybe just add a comment about why it isn't `.isParametric`, so we don't accidentally add that flag at some point